src/WarpedPsf.cc

```diff
@@ -69,14 +73,12 @@ PTR(afw::detection::Psf::Image) warpAffine(
     afw::detection::Psf::Image const &im, afw::geom::AffineTransform const &t,
     afw::math::WarpingControl const &wc
 ) {
-    static const int dst_padding = 0;
-
     afw::geom::AffineXYTransform xyTransform(t);
 
     afw::math::SeparableKernel const& kernel = *wc.getWarpingKernel();
     afw::geom::Point2I const& center = kernel.getCtr();
     int const xPad = std::max(center.getX(), kernel.getWidth() - center.getX());
-    int const yPad = std::min(center.getY(), kernel.getHeight() - center.getY());
+    int const yPad = std::max(center.getY(), kernel.getHeight() - center.getY());
```

Comment:
I've written a unit test  for this and will move the discussion to the Jira ticket, because I want to attach pictures: https://jira.lsstcorp.org/browse/DM-8175